src/routes/(dashboard)/ManagerDashboard.svelte

```diff
@@ -47,15 +47,9 @@
         { runLoadAgainWhenParamsChange: false }
     );
 
-    const searchParamsForUserWordCount = searchParameters(
-        {
-            sort: ssp.string(SortName.Days),
-            tab: ssp.string(Tab.myWork),
-            assignedUserId: ssp.number(0),
-            project: ssp.string(''),
-        },
-        { runLoadAgainWhenParamsChange: false }
-    );
```

Comment:
This was essentially overwriting the search params when the user interacted with sort on the user word count table. It feels like overkill to track that table's state in the search params so I moved sort handling to component state instead.